SimPEG/DCIP/DCIPUtils.py

```diff
@@ -179,7 +179,7 @@ def plot_pseudoSection(DCsurvey, axs, stype):
         Input:
         :param d2D, z0
         :switch stype -> Either 'pdp' (pole-dipole) | 'dpdp' (dipole-dipole)
-
+        :switch dtype=-> Either 'appr' (app. res) | 'appc' (app. con) | 'volt' (potential)
         Output:
         :figure scatter plot overlayed on image
 
```

Comment:
time and author meta data (ie the fact that it was edited on Feb 17) don't need to be included in the header. All of this is tracked on github (and github will likely do a better job keeping up to date with it than we will!)